docs/pages/2019_MARVEL_Psik_MaX/sections/bands.rst

```diff
@@ -14,17 +14,17 @@ calculation steps, are performed automatically by the WorkChain.
 
 The demonstration of the workchain will be performed in a Jupyter
 notebook. To run it, follow the instructions that were given for the
-querybuilder notebook in section [sec:querybuilder]. The only difference
+querybuilder notebook in section :ref:`querybuilder`. The only difference
 is that instead of selecting the notebook in the ``querybuilder``
 directory, go to ``pw/bandstructure`` instead and choose the
 ``bandstructure.ipynb`` notebook. There you will find some example
 structures that are loaded from COD, through the importer integrated
 within AiiDA. Note that the required time to calculate the bandstructure
-for these example structures ranges from 3 minutes to almost half an
+for these example structures ranges from 6 minutes to almost half an
 hour, given that the virtual machine is running on a single core with
 minimal computational power. It is not necessary to run these examples
 as it may take too long to complete. For reference, the expected output
-band structures are plotted in Fig.[fig:workchainbandstructures].
+band structures are plotted below:
```

Comment:
Here you can use `:numref:`, see the `workflows.rst` file for examples.
You have to give a reference label to the figure as well, but then you get a nice `Fig. 1` auto numbered link

Reply:
Thanks. There was only a series of figures there, at the end of the file. Do you think a label is necessary?

Reply:
For this particular file no, but it will be if we need to refer to it from other parts of the text. I think it will be good practice to keep explicit references for all figures such that they will also get numbered labels